Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Physical lifestyle traits

### Changelog:
- Added traits (5 levels each - Faintly/Slightly/Moderately/Outstandingly/Immensely) for every physical lifestyle tree.


## Tests:


## How to test:
- You can find all traits in the in-game encyclopedia and/or by spending perks in corresponding lifestyle tree.
- Check if new tooltip in physical lifestyle about traits (near the lifestyle exp progressbar) works correctly.
- Check if traits and their acquisition works correctly.
